### PR TITLE
Apparmor, /dev/net/tun and bugfixes

### PR DESCRIPTION
**Pull request by stgraber**
Bunch of random bits that piled up during my current trip.

### CODE REVIEW

File: lxd/apparmor.go [hunk 1]
```diff
@@ -50,17 +50,234 @@ const NESTING_AA_PROFILE = `
...
+  deny /sys/fs?*{,/**} wklx,
+
+  # Special exception for cgroup namespaces
+%s
+
+  # user input raw.apparmor below here
+  %s
+
+  # nesting support goes here if needed
+%s
```
Comment (tych0) [inline lxd/apparmor.go]: The indentation here and above is off, is that intentional?

Comment (tych0) [inline lxd/apparmor.go]: Oh, it gets deleted in the next patch so it doesn't matter :)

Comment (stgraber) [inline lxd/apparmor.go]: I think I changed all that in a later commit so shouldn't really matter.

It was semi-intentional in that the cgns and nesting blobs were already indented while raw.apparmor isn't.

File: lxd/apparmor.go [hunk 2]
```diff
@@ -341,6 +338,8 @@ func runApparmor(command string, c container) error {
 // Ensure that the container's policy is loaded into the kernel so the
 // container can boot.
 func AALoadProfile(c container) error {
+	aaParserSupports("")
```
Comment (tych0) [inline lxd/apparmor.go]: What's this for?

Comment (stgraber) [inline lxd/apparmor.go]: Leftover debugging while I was adding that function :)

Removed now.

**Comment by stgraber** [comment]
@tych0 ping

**Pull request merged**